diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/comments/CommentsRule.kt

```diff
@@ -116,10 +116,31 @@ class CommentsRule(private val configRules: List<RulesConfig>) : Rule("comments"
         }
     }
 
+    /**
+     * This is a very rare case. We should check this cases for 4 things:
+     *
+     * 1. If it is a class/object at the beginning of the line
+     * 2. If it is a function
+     * 3. If it is import/package implementation
+     * 4. If it is }. This case is used when } goes after one space and it is closing class or fun
+     */
+    private fun isCodeAfterCommentStart(text: String): Boolean {
+        val textWithoutCommentStartToken = text.removePrefix("//").trim()
+        return textWithoutCommentStartToken.contains(classRegex) ||
+                textWithoutCommentStartToken.contains(importOrPackageRegex) ||
+                textWithoutCommentStartToken.contains(functionRegex) ||
+                textWithoutCommentStartToken.contains(rightBraceRegex)
```

Comment:
You could create a list of the four regex in compnion object and use `any` here

Reply:
Yeah. That seems better